sdk/monitor/opentelemetry-exporter-azuremonitor/tests/test_base_exporter.py

```diff
@@ -0,0 +1,417 @@
+    #     self.assertEqual(len(os.listdir(exporter.storage.path)), 0)
+
+    # def test_transmission_206_bogus(self):
+    #     exporter=BaseExporter(
+    #         storage_path=os.path.join(TEST_FOLDER, self.id())
+    #     )
+    #     envelopes_to_export=map(lambda x: x.as_dict(), tuple(
+    #         [TelemetryItem(name="", time="")]))
+    #     exporter.storage.put(envelopes_to_export)
+    #     with mock.patch("requests.post") as post:
+    #         post.return_value=MockResponse(
+    #             206,
+    #             json.dumps(
+    #                 {
+    #                     "itemsReceived": 5,
+    #                     "itemsAccepted": 3,
+    #                     "errors": [{"foo": 0, "bar": 1}],
+    #                 }
+    #             ),
+    #         )
+    #         exporter._transmit_from_storage()
+    #     self.assertIsNone(exporter.storage.get())
+    #     self.assertEqual(len(os.listdir(exporter.storage.path)), 0)
+
+    # def test_transmission_400(self):
+    #     exporter=BaseExporter(
+    #         storage_path=os.path.join(TEST_FOLDER, self.id())
+    #     )
+    #     envelopes_to_export=map(lambda x: x.as_dict(), tuple(
+    #         [TelemetryItem(name="", time="")]))
+    #     exporter.storage.put(envelopes_to_export)
+    #     with mock.patch("requests.post") as post:
+    #         post.return_value=MockResponse(400, "{}")
+    #         exporter._transmit_from_storage()
+    #     self.assertEqual(len(os.listdir(exporter.storage.path)), 0)
+
+    # def test_transmission_439(self):
+    #     exporter=BaseExporter(
+    #         storage_path=os.path.join(TEST_FOLDER, self.id())
+    #     )
+    #     envelopes_to_export=map(lambda x: x.as_dict(), tuple(
+    #         [TelemetryItem(name="", time="")]))
+    #     exporter.storage.put(envelopes_to_export)
+    #     with mock.patch("requests.post") as post:
+    #         post.return_value=MockResponse(439, "{}")
+    #         exporter._transmit_from_storage()
+    #     self.assertIsNone(exporter.storage.get())
+    #     self.assertEqual(len(os.listdir(exporter.storage.path)), 1)
+
+    # def test_transmission_500(self):
+    #     exporter=BaseExporter(
+    #         storage_path=os.path.join(TEST_FOLDER, self.id())
+    #     )
+    #     envelopes_to_export=map(lambda x: x.as_dict(), tuple(
+    #         [TelemetryItem(name="", time="")]))
+    #     exporter.storage.put(envelopes_to_export)
+    #     with mock.patch("requests.post") as post:
+    #         post.return_value=MockResponse(500, "{}")
+    #         exporter._transmit_from_storage()
+    #     self.assertIsNone(exporter.storage.get())
+    #     self.assertEqual(len(os.listdir(exporter.storage.path)), 1)
+
+    # def test_transmission_empty(self):
+    #     exporter=BaseExporter(
+    #         storage_path=os.path.join(TEST_FOLDER, self.id())
+    #     )
+    #     status=exporter._transmit([])
+    #     self.assertEqual(status, ExportResult.SUCCESS)
```

Comment:
do we need this commented code anymore?

Reply:
Might be good to keep until we have the tests changed to use the client instead (for "TODO" and also we can see all the things we are trying to test).